integrated_channels/lms_utils.py

```diff
@@ -51,5 +62,25 @@ def get_single_user_grade(course_id, user):
     if not CourseGradeFactory:
         return None
     course_key = CourseKey.from_string(course_id)
-    course_grade = CourseGradeFactory().read(user.username, course_key=course_key)
+    course_grade = CourseGradeFactory().read(user, course_key=course_key)
     return course_grade
+
+
+def get_course_details(course_id):
```

Comment:
I need to add a test for this method, will get that in soon

Reply:
Looks like you have added one, resolving this comment.